src/test/java/org/broadinstitute/hellbender/utils/plotter/AlleleFractionSegmentedPlotterTest.java

```diff
@@ -8,7 +8,7 @@
 import java.io.File;
 import java.io.IOException;
 
-public class AlleleFractionPlotterTest extends BaseTest {
+public class AlleleFractionSegmentedPlotterTest extends BaseTest {
     @Test
     public void testAlleleFractionPlotting() throws IOException {
```

Comment:
Tried to do this and IntelliJ realized that `runCommandLine` never throws an `IOException`, so the `throw` was silly to begin with.  Removed -- and now the code looks like all the other integration tests in the repo which seem to throw only when there's an expected exception.